pygmt/tests/test_grdimage.py

```diff
@@ -205,10 +201,11 @@ def test_grdimage_central_meridians(grid, proj_type, lon0):
 # Cylindrical Equidistant (Q) projections plotted with xarray and NetCDF grids
 # are still slightly different with an RMS error of 25, see issue at
 # https://github.com/GenericMappingTools/pygmt/issues/390
-# TO-DO remove tol=1.5 and pytest.mark.xfail once bug is solved in upstream GMT
-@check_figures_equal(tol=1.5)
+# TO-DO remove pytest.mark.xfail once bug is solved in upstream GMT
+#       combine with test_grdimage_central_meridians_and_standard_parallels_fix
+@check_figures_equal()
```

Comment:
Need to put back the tolerance I think, still some small diffs resulting in test failures for the General Stereographic (S) projection plots


```python-traceback
FAILED pygmt/tests/test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[S-0-0-png] - pygmt.exceptions.GMTImageComparisonFailure: images not close...
FAILED pygmt/tests/test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[S-0-30-png] - pygmt.exceptions.GMTImageComparisonFailure: images not clos...
FAILED pygmt/tests/test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[S-123-30-png] - pygmt.exceptions.GMTImageComparisonFailure: images not cl...
FAILED pygmt/tests/test_grdimage.py::test_grdimage_central_meridians_and_standard_parallels[S-180-30-png] - pygmt.exceptions.GMTImageComparisonFailure: images not cl...
```

```suggestion
# TO-DO remove tol=1.5 and pytest.mark.xfail once bug is solved in upstream GMT
@check_figures_equal(tol=1.5)
```